cookbooks/bcpc-hadoop/recipes/hbase_config.rb

```diff
@@ -149,9 +149,10 @@
 env_sh = {}
 env_sh[:HBASE_PID_DIR] = '"/var/run/hbase"'
 env_sh[:HBASE_LOG_DIR] = '"/var/log/hbase"'
-env_sh[:HBASE_OPTS] = '" -Djava.net.preferIPv4Stack=true -XX:+UseConcMarkSweepGC"'
+env_sh[:HBASE_OPTS] = '" -Djava.net.preferIPv4Stack=true -XX:+UseConcMarkSweepGC" -Djava.security.auth.login.config=/etc/hbase/conf/hbase-client.jaas'
```

Comment:
Is it better if we add these env parameters when the cluster is only Kerberized like [this](https://github.com/ekund/chef-bach/blob/602d13ad822cb7f32173bdffdb8ede97c7f19522/cookbooks/bcpc-hadoop/recipes/hbase_config.rb#L85)